src/app/pages/rpc-sign-psbt/use-rpc-sign-psbt.tsx

```diff
@@ -31,8 +31,7 @@ export function useRpcSignPsbt() {
   const analytics = useAnalytics();
   const navigate = useNavigate();
   // const { addressNativeSegwitTotal, addressTaprootTotal, fee } = usePsbtSignerContext();
```

Comment:
Can we deleted this commented code while we're here? 

```
// const { addressNativeSegwitTotal, addressTaprootTotal, fee } = usePsbtSignerContext();
```

Reply:
Yep, thought I did tbh, weird. Thx for pointing out.